src/main/resources/templates/assessment-details.html

```diff
@@ -54,28 +54,30 @@ <h1 class="navbar-brand">Catalyst</h1> <!-- Company title -->
         <h2 class="mb-4">Visual Assessment's Details</h2>
         <button type="button" class="btn-close" onclick="history.back()" aria-label="Close"></button>
     </div>
-    <form method="post" id="viewTemplate"> <!---TODO th:action and th:object setup ---->
+    <form th:action="@{/assessments/{id}(id=${assessment.id})}" th:object="${assessment}" method="post"
+          id="viewTemplate1">
+        <!--    <form method="post" id="viewTemplate">-->
         <div class="row">
             <!-- Patient -->
             <div class="col-md-6 mb-3">
                 <label for="select-assessments-patients-field" class="form-label">Patient</label>
-                <!---TODO th:field (th:field="*patient) <-- need a getter method to get name? or change it instead of id get the patient object" ---->
-                <select class="form-select" id="select-assessments-patients-field">
-                    <option value="" selected disabled>Please select a patient</option> <!--will not get the list because it's not a proper template yet -->
+                <select class="form-select" id="select-assessments-patients-field" th:field="*{patient}">
+                    <option value="" selected disabled>Please select a patient</option>
+                    <!--will not get the list because it's not a proper template yet -->
                     <option th:each="patient : ${patients}"
                             th:value="${patient.id}"
-                            th:text="${patient.firstName + ' ' + patient.lastName + ': ID ' + patient.id}"></option>
```

Comment:
We should include the ID as there can be more than two clients with the same name